### PR TITLE
Upgrade preview window to use .asWebviewUri

### LINKED ISSUE #170: Content security policy
From the debug console:

> cesium.gltf-vscode created a webview without a content security policy: https://aka.ms/vscode-webview-missing-csp

### LINKED ISSUE #183: Adopt VS Code's 'asWebviewUri' API
Hi, I maintain VS Code's Webview API

## Issue
Our telemetry suggests that your extension uses webviews and may be loading resources in these webviews using hardcoded `vscode-resource:` URIs. These URIs have some important limitations and don't work properly when VS Code is run in a browser. We are also making changes in the desktop version of VS Code that may cause these URIs to not work properly in future versions of VS Code.

While we are making our best effort to continue support existing webview extensions that use `vscode-resource:` URIs on desktop versions of VS Code, we will not able to fully support all uses cases.

## Fix
To ensure that your extension continues to work in as many environments as possible—including on web and remotely—please:

- Switch to use the [`Webview.asWebviewUri` function](https://github.com/microsoft/vscode/blob/307cb32f306a527a30a5b1756e8bf548c141192c/src/vs/vscode.d.ts#L6703) for loading resources.

- Switch to use the [`Webview.cspSource` property](https://github.com/microsoft/vscode/blob/307cb32f306a527a30a5b1756e8bf548c141192c/src/vs/vscode.d.ts#L6714) in content security policies.

These APIs shipped around 2 years ago so they should be available in all modern versions of VS Code. You can find additional information about the issue here: https://github.com/microsoft/vscode/issues/97962

Let me know if you have any questions about this change

### LINKED ISSUE #184: Previewing glTF shows a blank screen
When I try to preview a glTF I am seeing a blank screen (See screenshot). It used to work fine, but I think there was a recent VS Code update that might be causing the issue. Does anyone know why this would happen?
glTF-Viewer

### PR DESCRIPTION
**Pull request by emackey**
Fixes #170
Fixes #183
Fixes #184

### CODE REVIEW

**Comment by emackey** [comment]
Version `2.2.11` has been published with these changes.

@bghgary I published this in a hurry because the preview window was completely broken in recent VSCode versions.  But once you're back and have time for a more careful review, I'd be glad to make any further changes to this.

**Comment by bghgary** [comment]
These changes seem reasonable to me.

**Comment by emackey** [comment]
Thanks!

File: pages/babylonView.html [hunk 1]
```diff
@@ -1,2 +1,2 @@
 <canvas id="babylonRenderCanvas"></canvas>
-<a href="http://babylonjs.com/"><img id="babylonLogo" src="{extensionRootPath}engines/Babylon/logo.svg"></img></a>
```
Comment (bghgary) [inline pages/babylonView.html:2]: Maybe also remove the logo file for Babylon?

**Pull request merged**